server/pulp/server/async/manage_workers.py

```diff
@@ -17,11 +17,12 @@
 After=network.target
 
 [Service]
+UMask=0002
 EnvironmentFile=%(environment_file)s
 User=apache
 WorkingDirectory=/var/run/pulp/
 ExecStart=/usr/bin/celery worker -n reserved_resource_worker-%(num)s@%%%%h -A pulp.server.async.app\
-          -c 1 --events --umask 18 --pidfile=/var/run/pulp/reserved_resource_worker-%(num)s.pid\
+          -c 1 --events --pidfile=/var/run/pulp/reserved_resource_worker-%(num)s.pid\
```

Comment:
I haven't figured out why yet, but the --umask param to celery appears to have no impact on the actual umask used. I removed the param here, rathe rthan have it (confusingly) being set in multiple places.

Reply:
Answering myself here: per the celery CLI docs, "--umask : Effective `umask(1)` (in octal) of the process **after detaching**." We weren't setting in octal, and in any event, it didn't matter, because we do not specify `-D/--detach`, so there is no detached-process to set the umask for and the parameter is ignored.